### ISSUE TITLE
Copy, Paste and Cut don't work

### ISSUE THREAD

**Issue by jeanlucdegruchy** [Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker#780]
The tutorials and key controls tell me that in order to copy and paste you must use CTRL+C then CTRL+V but when I use those commands nothing happens. At first I thought it was just during multi-select but it doesn't work at all with any tiles or props. I have restarted the board and even opened a new one and the problem persists.

The undo and redo commands (CTRL+Z and CTRL+Y) work perfectly fine, the problem only appears to be with copying, pasting and cutting.

I have tried using a different keyboard, uses both CTRL keys and even tried altering the commands in settings to Alt S, CTRL S, Shift M etc but nothing happens when i do these functions.

When I press CTRL+C in order to copy it thinks I am just pressing C for the eyedropper "pick an asset" function.

Please help!

**Comment by Ja07sj** [comment]
Can you try holding CTRL+C and CTRL+P for like half a second longer when pressing? I ran into issues in earlier builds where i was pressing too quickly, and couldn't get the commands to register in the client.

**Comment by jeanlucdegruchy** [comment]
I have attempted to hold down the keys for several seconds and nothing registered.

**Comment by LuPro** [comment]
Are you in build mode while using copy and paste? Additionally, for copy to be able to copy something you have to select it with the volume select tool first (can't copy anything if nothing is selected).

**Comment by jeanlucdegruchy** [comment]
Yes I am in build mode and it doesn't copy when I'm using the volume select tool. I make sure that everything is within the cube and I follow the tutorials by left clicking to confirm the volume. But it won't copy.

**Comment by jectordelan** [comment]
I have to hit the x or c button numerous times to get the cut or copy to work. So don't just hit it once, hold ctrl down and hit x three or four times until it cuts.

**Comment by Funggiss** [comment]
My observation is that holding ctrl+c ctrl+x or ctrl+z doesn't work but tapping both keys quickly works.

**Comment by FlameRose** [comment]
I am having the same issue.  Tapping them both quickly worked the other day, but is not consistent.  I cannot consistently copy paste anything I made or slabs.  I CAN add boards and paste into the text box.  I am on Windows 10 and I disconnected my offhand controller to make sure it was not interfering.  I was in the Discord channel and troubleshooting for about two hours with no luck.
